graphql/participant.js

```diff
@@ -29,3 +30,15 @@ ParticipantTC.addRelation(
     projection: { userId: true },
   }
 );
+
+ParticipantTC.addRelation(
+  'kudo',
+  {
+    resolver: KudoTC.getResolver('findMany'),
+    prepareArgs: {
+      giverId: source => source.giverId,
```

Comment:
I need to check if that works, I think there should be something like `receiverId: source => source.user_id` and `giverId: () => currentUser._id`

Reply:
context is third argument (source, args, context) => context.id

Reply:
and I think they should be wrapped in **filter** key